cli/src/pcluster/config/mappings.py

```diff
@@ -95,7 +95,6 @@
     maintain_initial_size_validator,
     queue_settings_validator,
     queue_validator,
-    raid_volume_iops_validator,
```

Comment:
let's remove this validator since now is unused

Reply:
Done!